src/Sarif.Sarifer/Sarif.Sarifer.csproj

```diff
@@ -51,6 +51,11 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.VSSDK.BuildTools">
+      <Version>17.0.38-dev17-g08fcf558</Version>
```

Comment:
>17.0.38-dev17-g08fcf558 [](http://example.com/codeflow?start=15&length=23)

the new version is available: [https://www.nuget.org/packages/Microsoft.VSSDK.BuildTools/](https://www.nuget.org/packages/Microsoft.VSSDK.BuildTools/)

Reply:
you can  also remove the nuget.config

Reply:
removed

Reply:
updated to official version

---
In reply to: [637965164](https://github.com/microsoft/sarif-visualstudio-extension/pull/397#discussion_r637965164) [](ancestors = 637965164)